src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/ManageOrders.java

```diff
@@ -286,4 +276,25 @@ public class ManageOrders implements MappableObject {
     //PRL-4216 - serve order additional documents
     @JsonProperty("additionalOrderDocuments")
     private List<Element<AdditionalOrderDocument>> additionalOrderDocuments;
+
```

Comment:
Please add a comment to indicate if any no longer fields needs to be removed in future